client/src/shared/services/documentService.js

```diff
@@ -6,14 +6,18 @@ export default angular.module('shared')
         function getPopularDocuments() {
             return $http.get(`${SERVER_URL}/documents/popular`);
         }
-        function searchDocuments(query, filtersArray) {
+        function searchDocuments(query, filtersArray, countFilters) {
```

Comment:
Please, rename variable to isCountFilters to underlain boolean.

Reply:
fixed